besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1284,6 +1284,12 @@ static class TxPoolOptionGroup {
           "Specifies the static node file containing the static nodes for this node to connect to")
   private final Path staticNodesFile = null;
 
+  @CommandLine.Option(
+      names = {"--rpc-max-logs-range"},
```

Comment:
are we only adding this limit for HTTP (not WS?) - if so let's call it rpc-http-max-logs-range - similar to rpc-http-max-active-connections

Reply:
It's not specific to HTTP - it should be effective across HTTP, WS and IPC.